src/app/(school)/page.tsx

```diff
@@ -1,6 +1,7 @@
 "use client"; // Remove this when we remove api calls directly in page
-
+import { ReviewSection } from "@/common/components/ReviewSection/ReviewSection";
```

Comment:
suggestion: add `index.ts` at `@/common/components/ReviewSection` to export the various exported vars to eliminate 1 extra level of importing

Reply:
fixed https://github.com/AfterClass-io/afterclass.io-v2/pull/104/commits/7cdf46559f65df476ed690af89d3dedc6d9716c8